doc.txt

```diff
@@ -34,7 +34,8 @@
  * The following list of what `DEVELHELP=1` enables is not comprehensive, but
  * should give a rough impression of what to expect:
  *
- * * Many runtime checks are enabled (stack overflow protection, warnings when
+ * * Many runtime checks are enabled (stack overflow protection by means of
+ *   @ref pseudomodule_mpu_stack_guard or @ref SCHED_TEST_STACK, warnings when
```

Comment:
Hmm for some reason `SCHED_TEST_STACK` not defined, I think the `#if defined(DEVELHELP) || defined(DOXYGEN)` needs to be only around the define and not the comment, can you try that?

Reply:
Indeed; that was fixed in https://github.com/RIOT-OS/RIOT/pull/17132, and the hardships we had finding it was one of the things that triggered this.

I've applied @asbadie's suggestion, and will squash-rebase onto master to get the defined SCHED_TEST_STACK in.